backend/geonature/core/gn_synthese/utils/taxon_sheet.py

```diff
@@ -62,3 +62,20 @@ def get_area_subquery(area_type: str) -> Query:
             .where(LAreas.id_type == BibAreasTypes.id_type, BibAreasTypes.type_code == area_type)
             .alias("areas")
         )
+
+    @staticmethod
+    def get_taxon_subquery(cd_ref: int) -> Query:
```

Comment:
```suggestion
    def get_taxon_subquery(cd_ref: int) -> Select:
```
Voir https://docs.sqlalchemy.org/en/14/core/selectable.html#sqlalchemy.sql.expression.Select

Reply:
ok, je change le nom de la méthode au passage